src/components/MobileNavBar/NavBar.storybook.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { RouterDecorator } from '../../RouterDecorator';
+import { MobileNavBar } from '.';
+
+storiesOf('Mobile NavBar', module)
+  .addDecorator(RouterDecorator)
```

Comment:
Since this component is only ever meant to be displayed on mobile devices, we should add the `withViewport` decorator from `@storybook/addon-viewport` so we can default the viewport to a mobile one. You can see the documentation at https://github.com/storybooks/storybook/tree/master/addons/viewport#withviewport-decorator

Reply:
I was actually thinking about that, but there's no type def library for storybook viewport, and only the alpha version (aka master on github) has `withViewport()`. Ik that's a lot of excuses, but do you think it's worth working around having the type defs?

Reply:
I think it's worth working around the typedefs. If we need to present the mobile components to the client, it'll be much easier if we can show them the component within a mobile viewport by default, as opposed to requiring them to open up the addon and select a device.